### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key at Navigation screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#826]
**Describe the bug**
Web page is not being navigated sequentially with the Tab key at Navigation screen. The sequence of focus is unordered.

**To Reproduce**
1. Go to Navigation screen.
2. Use Tab key to focus on different components of the screen.
3. Focus is not moving in sequence and is not going to "Delete Site", "Copy Site", "Move Section" buttons, etc.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
@natechadwick 

Hi Nate, I analyzed it and it seems that for  perclisteditorwidget.js we can't  have same  tabindex sequencing.

**Comment by ramdev1980** [comment]
We have to set the tabindex=0 for this screen.
